README.md

```diff
@@ -254,6 +255,46 @@ export default class CounterApp {
 }
 ```
 
+#### Selectors
+
+Selectors let you define views on your state. They enable you to define derived data on your store's state. 
+In combination with memoized functions the calculation overhead can be minmized. Hence, the child components of a Connector 
+ using memoized selectors will only be rerendered if the source data of the selector changes. This can help to
+ prevent store dependencies.
+ 
+It is further recommended to define complex selectors in separate modules.
+
+e.g. defining selectors for a Todo Store
+```js
+import { createSelector, createBuffered } from 'redux';
+
+export let todoSelector = createSelector('todos');
+export let numberOfTodos = createSelector(todoSelector, createBuffered(todos => todos.length)); 
+```
+
+using the selector in your Component
+```js
+import React from 'react';
+import { Connector } from 'redux/react';
+//import your selectors
+import { numberOfTodos } from 'selectors/TodoSelectors';
+
+export default class TodoCount {
+  render() {
+    return (
+      //use your predefined selectors
+      <Connector select={state => ({
```

Comment:
What if `select` actually took selectors in the same format as nuclear-js getters, that is an array. So this would be:

```js
select={['todos', todos => todos.length]}
```

You could still import an array selector from somewhere else, and testing it would require you to call `createSelector`, but it would allow the individual connector to worry about memoziation (not sure if this is better or not...)